test/e2e/crd.go

```diff
@@ -98,8 +98,9 @@ func validateCrdCrud(f framework.FederationFramework, targetCrdKind string, name
 			},
 		},
 	})
+	shortNames := []string{"tapir"}
 
-	targetCrd := federate.CrdForAPIResource(targetAPIResource, validationSchema)
+	targetCrd := federate.CrdForAPIResource(targetAPIResource, validationSchema, shortNames)
```

Comment:
I think it's preferable to pass nil here, since short names don't figure in this test.

Reply:
I was planning to update the validation once I figured out where it was validated. But, the test error didn't help me out.